src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,35 +37,37 @@ export const TodoModal: React.FC = () => {
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              {`Todo #${selectedTodo.id}`}
             </div>
 
             {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
             <button
               type="button"
               className="delete"
               data-cy="modal-close"
+              onClick={handleCloseButton}
             />
           </header>
 
           <div className="modal-card-body">
             <p className="block" data-cy="modal-title">
-              quis ut nam facilis et officia qui
+              {selectedTodo.title}
```

Comment:
```suggestion
              {title}
```

Reply:
But, how ? ...
if it comes from selectedTodo...
Please, explain

Reply:
Use destructuring assignment 
const { id, title, ... } = selectedTodo